backend/src/main/java/com/benine/backend/Main.java

```diff
@@ -40,9 +41,28 @@ public static void main(String[] args) {
 
     // Setup camerahandler
     cameraController = new CameraController();
+    
+    //Temporary adding multiple test camera's
     SimpleCamera camera = new SimpleCamera();
-    camera.setStreamLink("tuincam.bt.tudelft.nl/mjpg/video.mjpg");
+    camera.setStreamLink("http://tuincam.bt.tudelft.nl/mjpg/video.mjpg");
     cameraController.addCamera(camera);
+    Camera camera2 = new IPCamera2("83.128.144.84:88");
+    cameraController.addCamera(camera2);
+    SimpleCamera camera3 = new SimpleCamera();
+    camera3.setStreamLink("http://131.180.123.51/cgi-bin/nph-zms?mode=jpeg&monitor=4&scale=100&maxfps=6&buffer=100");
+    cameraController.addCamera(camera3);
```

Comment:
Please get stuff like this from the config, it makes the code way more readable.